<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTaskDialog</class>
 <widget class="QDialog" name="ViewTaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Task Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Due Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Details:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Month:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Day:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="monthComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="dayComboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="timeComboBox1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="timeComboBox2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskNameLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
